### PR TITLE
[CLOUD-2797] - update jolokia config to support both old and new JSON modules

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-2797
Signed-off-by: Ken Wills <kwills@redhat.com>

### CODE REVIEW

**Comment by rcernich** [comment]
Ironically, there's another JIRA suggesting a removal of bootclasspath, because it breaks running with SecurityManager.  See CLOUD-2073.

**Comment by bstansberry** [comment]
@luck3y Can you try a variant of this that just removes the -Xbootclasspath stuff altogether?  Per Rob's comment on CLOUD-2073 he thinks it's not needed any more.

**Comment by luck3y** [comment]
@bstansberry I experimented with taht a little yesterday, but forgot to comment. Removing it completely throws a CNFE on the log class. 

I removed everything else after that to see what would happen (jolokia and hawklular etc), and still got the same exception. (I'll retest and paste the exception here.)

Looking at the related tickets, it suggested that JAVA_OPTS_APPEND=-Dsun.util.logging.disableCallerCheck=true was required, which I haven't tried.

tl;dr version, yep, will retest :)

**Comment by luck3y** [comment]
I'm going to go ahead and merge this, so the dev branch is usable for WF 14 / snapshot builds again.
